### PR TITLE
add `mutedErrors` itinerary config feature

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**

Adds a new config feature that allows the OTP2 error renderer to be disabled for errors a config does not want to display.

Co-authored by @AdrianaCeric 

**PR Checklist:**
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by amy-corson-ibigroup — APPROVED** [review]
Works great! Thank you!

File: lib/components/narrative/metro/metro-error-renderer.tsx [hunk 1]
```diff
@@ -57,6 +65,9 @@ const ErrorRenderer = ({ errors }: { errors: Error }): JSX.Element => {
           return null
         }
 
+        // Don't show errors that have been muted in the config
```
Comment (binh-dam-ibigroup) [inline lib/components/narrative/metro/metro-error-renderer.tsx:68]: Maybe move the two `return null` to the top of the mapping function.

**Review by binh-dam-ibigroup — APPROVED** [review]
Should be good to go after resolving conflicts.

File: lib/components/narrative/metro/metro-error-renderer.tsx (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/components/narrative/metro/metro-error-renderer.tsx:61]: Good catch thanks!

**Pull request merged**